src/types.ts

```diff
@@ -15,6 +15,11 @@ export interface Zone {
   resources: Resource[];
 }
 
+export interface ZoneLight {
```

Comment:
Why a separate type? It is only used at compile time.

Reply:
If you don't have all the info available for a full Zone, then move this type to where it is actually relevant, not as a global type.

Reply:
Yeah, I could probably get the existing ones to work the way I need them too. I was actually racking my brain trying to figure out this problem for a while until an ah-ha moment. I never went back and reiterated on it after the fact. Plus I wanted to test it out with my guild for a bit. 

In retrospect, this should have been a draft PR.